shared_model/builders/protobuf/common_objects/proto_account_asset_builder.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_PROTO_ACCOUNT_ASSET_BUILDER_HPP
+#define IROHA_PROTO_ACCOUNT_ASSET_BUILDER_HPP
+
+#include "backend/protobuf/common_objects/account_asset.hpp"
+#include "responses.pb.h"
+#include "utils/polymorphic_wrapper.hpp"
+
+namespace shared_model {
+  namespace proto {
+    /**
+     * AccountAssetBuilder is used to construct AccountAsset proto objects with initialized
+     * protobuf implementation
+     */
+    class AccountAssetBuilder {
```

Comment:
Rework with `CommonBuilder`

Reply:
CommonBuilder is used for interface builders, I do not know how we can apply it to proto builders